ztp/resource-generator/entrypoints/generator

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/env bash
+
+PROG=$(basename $0)
+SUB_COMMAND=$1
+
+# sub command-line arguments
+EMS_ONLY="false"
+WRAP_IN_POLICY="true"
+
+ztpTmpDir="/home/ztp/tmp"
+basePath="/kustomize/plugin/ran.openshift.io/v1"
+policyGenPath=${basePath}/policygentemplate
+siteConfigPath=${basePath}/siteconfig
+
+printenv MOUNT_DIR &> /dev/null
+if [[ $? -ne 0 ]]; then
+  # default dest dir in the container that is mounted from the host
+  MOUNT_DIR="/home/ztp_resources"
+fi
+
+function help {
+    cat >&2 <<EOF
+ztp-site-generator single node resources generator
```

Comment:
Maybe this should be "single site" instead of "single node" since this will work for multi-node clusters as well!